lib/pakyow/app/assets/console/styles/_data.scss

```diff
@@ -112,16 +112,20 @@
     background: $color-blue;
     position: absolute;
     color: $color-white;
-    margin-top: -37px;
+    margin-top: -36px;
     margin-left: 100px;
     border-top-right-radius: $radius;
     border-bottom-right-radius: $radius;
     @include rem(font-size, 14);
-    padding: 10px;
+    padding: 11px;
     padding-top: 11px;
     border-left: 1px solid darken($color-blue, 10%);
     cursor: pointer;
 
+    @include mobile {
```

Comment:
This is why I changed the last 3 items because this is how it looked before I changed anything. 
<img width="962" alt="screen shot 2017-02-09 at 7 04 39 am" src="https://cloud.githubusercontent.com/assets/8171067/22784576/2eedb44e-ee96-11e6-890f-1b4f9f772ca4.png">
<img width="354" alt="screen shot 2017-02-09 at 7 04 18 am" src="https://cloud.githubusercontent.com/assets/8171067/22784581/32b39a58-ee96-11e6-8b3e-e5e200122870.png">